client/landing/gutenboarding/index.tsx

```diff
@@ -18,6 +18,7 @@ import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
 import config from '../../config';
 import { subscribe, select, dispatch } from '@wordpress/data';
 import { initializeAnalytics } from '@automattic/calypso-analytics';
+import type { Site, User } from '@automattic/data-stores';
```

Comment:
Would it make sense (and is it possible) to alias these here so we don't override them [later](https://github.com/Automattic/wp-calypso/pull/42804/files#diff-81026c81e09dfd61cbf4557a93277a1aR59-R60)?

Reply:
That doesn't look good! Thanks for spotting.

That caused no errors, I'm not sure why exactly. `Site` may be a namespace/module, so didn't import a type which meant that the `Site` type name was available?

Pushed a fix so the names don't collide.

Reply:
I think that was indeed it, namespace/module vs. type:

```ts
// React isn't a type, no error:
import type * as React from 'react'
type React = React.Component;

// Reducer is a type, this is an error:
import type { Reducer } from 'redux';
type Reducer = 'foo';
```

Good to know this would error if necessary!